lib/src/select/Option.tsx

```diff
@@ -40,7 +38,11 @@ const Option = ({
         grouped={isGroupedOption}
         multiple={multiple}
       >
-        {multiple && <DxcCheckbox checked={isSelected} tabIndex={-1} />}
+        {multiple && (
+          <div style={{ display: "flex", pointerEvents: "none" }}>
+            <DxcCheckbox checked={isSelected} tabIndex={-1} />
+          </div>
+        )}
```

Comment:
I don't agree with this. I think this is the cleanest solution. To manually control the focus and where it moves (I tried it) requires a lot of extra code to obtain basically the same result. Also, since the `<li>` is the one that has the interactive part there (hover, active), the checkbox is just a visual representation of the multiple-choice selection. Removing it's interactive part, for me, makes a lot of sense here.

Trying to find a "better" solution to just avoid a rule from the accessibility tool I don't think that's a good enough reason.

Reply:
It's actually not only about "avoiding" the rule, it actually makes a difference for assistive technologies, such as screen readers, which might ignore or respond unexpectedly to such nested controls.

I would like to investigate further about what the possible approaches are in order to prevent this scenario without needing to overcomplicate the current API for the `Checkbox` component.

Reply:
But, does the screen reader throw you any unexpected behaviour in this case? I mean, it just works as before.